bindings/pyroot_experimental/cppyy/CPyCppyy/src/Converters.cxx

```diff
@@ -2068,6 +2069,8 @@ static struct InitConvFactories_t {
         gf["const ULong64_t&"] =            gf["const unsigned long long&"];
 
     // factories for special cases
+        gf["TString"] =                     (cf_t)+[](long) { return new TStringConverter{}; };
+        gf["const TString&"] =              (cf_t)+[](long) { return new TStringConverter{}; };
```

Comment:
what about `"TString&"` and `TString*`? Do they need special cases?

Reply:
I kept the two ones that are present in the current PyROOT (`TString` and `const TString&`). I can easily add `TString&`, but `TString*` does not work with the converter macros for strings of Cppyy, and it is a more complex case (it can mean an array of `TString`).